### ISSUE TITLE
Cannot access exam after payment (URGENT)

### ISSUE THREAD

**Issue by johnedmiston** [danielneis/moodle-availability_paypal#43]
Students have to pay $7 by PayPal before they can access each exam. We use the PayPal Availability plug in for this. 

Some students pay the money, then cannot access the exam, they are told to pay again and sometimes pay twice, which upsets them, even after paying twice they cannot do the exam. This has now become an urgent issue.

When we log in as the student, the same thing happens to us. We have tried clearing cache, reloading the page and so on.

Moodle 3.11, running on Rocky Linux 9.3,  using Maria DB. 
 
We have screenshots if you want.

Cheers,
John Edmiston

**Comment by oceance** [comment]
Hi!

I have the same problem - any luck with this issue?

**Comment by johnedmiston** [comment]
No, still working on it, trying to get paypal merchat support to get back
to us, if you are the praying type, send up a prayer!!

Cheers,
John
